source/contracts/Controller.sol

```diff
@@ -120,30 +117,15 @@ contract Controller is IController {
     function transferOwnership(address _newOwner) public onlyOwnerCaller returns (bool) {
         // if in dev mode, blacklist old owner and whitelist new owner
         if (whitelist[owner]) {
-            whitelist[owner] = false;
+            removeFromWhitelist(owner);
             whitelist[_newOwner] = true;
```

Comment:
Suggest using `addToWhitelist(_newOwner)` instead of editing `whitelist` directly.